libs/transfer/src/lib/components/make-transfer-summary/make-transfer-summary.component.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
I miss a test checking that it shows a summary when a transfer is done. Can you add it, please?

Reply:
I changed because property 'title' comes from an index signature, so it must be accessed with ['title'].ts(4111)

I can disable noPropertyAccessFromIndexSignature if needed.

Reply:
ok